Makefile

```diff
@@ -8,6 +8,7 @@ IPHONEOS_DEPLOYMENT_VERSION=12.0
 MACOSX_DEPLOYMENT_TARGET=10.15
 XROS_DEPLOYMENT_VERSION=1.0
 APPLETVOS_DEPLOYMENT_VERSION=12.0
+WATCHOS_DEPLOYMENT_VERSION=2.0
```

Comment:
it says 2.0 here, then it's 6.0 in Xcode project, but also 8.0 in Project.swift

what is the deployment target is should use?

Reply:
```suggestion
WATCHOS_DEPLOYMENT_VERSION=8.0
```